example/cumulus-tf/main.tf

```diff
@@ -28,6 +28,7 @@ data "aws_region" "current" {}
 data "terraform_remote_state" "data_persistence" {
   backend = "s3"
   config  = var.data_persistence_remote_state_config
+  workspace = "${terraform.workspace}"
```

Comment:
This works if you don't specify a workspace right since terraform defaults to `default`?

Reply:
Correct. And do we need have a CHANGELOG entry for this, since this change is only in our example deployment and not the modules we provide to external users?

Reply:
Personally, I would argue this doesn't warrant a changelog entry, just like turning off PITR on our deployment didn't get a changelog entry, because it only affected our example deployments.

Reply:
Ok I see it's not actually enabling it for everyone. You guys are right. Will approve, thanks!